spec/javascripts/fixtures/learner_admin.html

```diff
@@ -0,0 +1,6 @@
+<select id="id_course_mode">
```

Comment:
@haikuginger That's not even 10% of learner_admin HTML :) Suggestion: either rename it (to keep it a minimal amount of markup for course_mode tests) or extend to have a more or less actual HTML.